frontend/src/index.css

```
@@ -20,108 +20,116 @@
-    @apply bg-background text-foreground;
+    @apply text-foreground;
+    background-color: var(--background);
+    background-image: linear-gradient(
+      135deg,
+      var(--gradient-page-start) 0%,
+      var(--gradient-page-end) 100%
+    );
+    background-attachment: fixed;
+    font-family:
+      'SUIT Variable',
+      'Inter',
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)